src/Services/AppInfoService.cs

```diff
@@ -2,22 +2,33 @@
 // Licensed under the MIT license.
 
 using System.Reflection;
+using System.Security.Principal;
 using DevHome.Common.Helpers;
 using DevHome.Common.Services;
 using DevHome.Helpers;
 using Windows.ApplicationModel;
 
 namespace DevHome.Services;
 public class AppInfoService : IAppInfoService
-{
+{
+    private static bool RunningAsAdmin

```

Comment:
I didn't know about this service! That's great!

Few suggestions:
- Make `RunningAsAdmin` public so the entire app can benefit from it. I know the setup flow uses this method today, so we can refactor that part after this change.
- Use `Lazy` so the value is cache the next time we call this method